bastionzero/service/policies/organizationcontrols.go

```diff
@@ -0,0 +1,141 @@
+func (s *PoliciesService) CreateOrganizationControlsPolicy(ctx context.Context, policy *OrganizationControlsPolicy) (*OrganizationControlsPolicy, *http.Response, error) {
+	u := orgControlsBasePath
+	req, err := s.Client.NewRequest(ctx, http.MethodPost, u, policy)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	p := new(OrganizationControlsPolicy)
+	resp, err := s.Client.Do(req, p)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return p, resp, nil
+}
+
+// GetOrganizationControlsPolicyPolicy fetches the specified OrganizationControlsPolicy policy
+//
+// BastionZero API docs: https://cloud.bastionzero.com/api/#get-/api/v2/policies/organization-controls/-id-
+func (s *PoliciesService) GetOrganizationControlsPolicyPolicy(ctx context.Context, policyID string) (*OrganizationControlsPolicy, *http.Response, error) {
+	u := fmt.Sprintf(orgControlsSinglePath, policyID)
+	req, err := s.Client.NewRequest(ctx, http.MethodGet, u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	policy := new(OrganizationControlsPolicy)
+	resp, err := s.Client.Do(req, policy)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return policy, resp, nil
+}
+
+// DeleteOrganizationControlsPolicy deletes the specified OrganizationControls policy
+//
+// BastionZero API docs: https://cloud.bastionzero.com/api/#delete-/api/v2/policies/organization-controls/-id-
+func (s *PoliciesService) DeleteOrganizationControlsPolicy(ctx context.Context, policyID string) (*http.Response, error) {
+	u := fmt.Sprintf(orgControlsSinglePath, policyID)
+	req, err := s.Client.NewRequest(ctx, http.MethodDelete, u, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := s.Client.Do(req, nil)
+	if err != nil {
+		return resp, err
+	}
+
+	return resp, nil
+}
+
+// ModifyOrganizationControlsPolicy updates a OrganizationControls policy. All user populated
+// fields are mutable except for policy.TimeExpires.
+//
+// BastionZero API docs: https://cloud.bastionzero.com/api/#patch-/api/v2/policies/organization-controls/-id-
+func (s *PoliciesService) ModifyOrganizationControlsPolicy(ctx context.Context, policyID string, policy *OrganizationControlsPolicy) (*OrganizationControlsPolicy, *http.Response, error) {
+	u := fmt.Sprintf(orgControlsSinglePath, policyID)
+	req, err := s.Client.NewRequest(ctx, http.MethodPatch, u, policy)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	p := new(OrganizationControlsPolicy)
+	resp, err := s.Client.Do(req, p)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return p, resp, nil
+}
+
+// Ensure OrganizationControlsPolicy implementation satisfies the expected interfaces.
+var (
+	// OrganizationControlsPolicy implements PolicyInterface
+	_ PolicyInterface = &OrganizationControlsPolicy{}
+)
+
+func (p *OrganizationControlsPolicy) GetPolicyType() policytype.PolicyType {
+	return policytype.OrganizationControls
+}
+
+func (p *OrganizationControlsPolicy) GetMfaEnabled() bool {
+	return p.mfaEnabled
+}
```

Comment:
```suggestion
func (p *OrganizationControlsPolicy) GetMFAEnabled() bool {
	if p.MFAEnabled == nil {
		return false
	}
	return *p.MFAEnabled
}
```

Add helper getter in case users want to just work with value type and not have to do nil pointer check (similar to other policy types). The getter should return the default value for the type if the pointer is nil